__test__/api/comments/updateSingleCommentFlags.test.js

```diff
@@ -0,0 +1,101 @@
+const app = require("../../../server");
+const CommentModel = require("../../../models/comments");
+const supertest = require("supertest");
+const request = supertest(app);
+
+describe("PATCH /comments/:commentId/flag", () => {
+  let sampleComment;
+
+  afterEach(async () => {
+    await CommentModel.findByIdAndDelete(sampleComment.commentId);
+    sampleComment = null;
+  });
+
+  test("200 - Flag a single comment", async () => {
+    const comment = new CommentModel({
+      refId: "5edd40c86762e0fb1287503",
+      applicationId: global.application._id,
+      ownerId: "myuseremail@email.com",
+      content: "this is a comment",
+      origin: "myuseremail@email.com",
+    });
+    await comment.save();
+    sampleComment = comment;
+    const res = await request
+      .patch(`/v1/comments/${comment._id}/flag`)
```

Comment:
check #357 or #372 for guidance, after updating the db via endpoint, you need to **also** check that the record from direct db search is the same, similar to:

```javacript
let x = PATCH /url/
let y = await CommentModel.findById(...)
x === y
```

Reply:
ok thanks for your explanation.